src/main/java/domain/Buyer.java

```diff
@@ -0,0 +1,49 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+public class Buyer {
+
+  private Money money;
+  private List<Lotto> lottos = new ArrayList<>();
+
+  public Buyer(Money money) {
+    this.money = money;
+    createLottos();
+  }
+
+  public int getLottoSize() {
+    return lottos.size();
+  }
+
+  public void createLottos() {
+    int totalLotto = money.calculateTotalLotto();
+    for (int i = 0; i < totalLotto; i++) {
+      lottos.add(new Lotto());
+    }
+  }
+
+  public String createResult() {
+    String result = "";
+    for (Lotto lotto : lottos) {
+      result += lotto.toString() + "\n";
+    }
+    return result;
+  }
+
+  public HashMap<LottoMatch, Integer> countLottos(WinningLotto winningLotto) {
+    HashMap<LottoMatch, Integer> result = new LinkedHashMap<>();
```

Comment:
마찬 가지로 변수의 타입도 상위 타입을 선언해보면 좋을 것 같습니다. 😃 
```suggestion
    Map<LottoMatch, Integer> result = new LinkedHashMap<>();
```